.github/workflows/cherry-picks.yml

```diff
@@ -0,0 +1,37 @@
+name: Cherry Pick to remote repository
+on:
+  pull_request_target:
+    types: [closed]
+permissions:
+  contents: write # so it can comment
+  pull-requests: write # so it can create pull requests
+jobs:
+  cross-repo-cherrypick:
+    name: Cherry pick to remote repository
+    runs-on: ubuntu-latest
+    # Don't run on closed unmerged pull requests
+    if: github.event.pull_request.merged
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.FINEGRAINED_PAT }}
+      - name: Create backport pull requests
+        uses: jschmid1/cross-repo-cherrypick-action@807f3f450c82fe6dd40220b1cee0449a3fb26b44
+        with:
+          token: ${{ secrets.FINEGRAINED_PAT }}
+          pull_title: '[cherry-pick -> ${target_branch}] ${pull_title}'
+          merge_commits: 'skip'
+          trigger_label: 'cherry-pick kong-ee' # trigger based on this label
```

Comment:
```suggestion
          trigger_label: 'cherry-pick-kong-ee' # trigger based on this label
```

Reply:
is there a specific reason you want to change this?

Reply:
Disagree. spaces are allowed in the tag. The 2 hyphens is needed for both words